reference/docs-conceptual/whats-new/Migrating-from-Windows-PowerShell-51-to-PowerShell-7.md

```diff
@@ -0,0 +1,326 @@
+the at sign ('@'), followed by the computer name.
+
+```powershell
+Enter-PSSession -HostName <Username>@<Computer>
+```
+
+You may set up SSH key authentication using a private key file with the **KeyFilePath** parameter.
+For more information, see [OpenSSH Key Management](/windows-server/administration/openssh/openssh_keymanagement).
+
+### Group Policy supported
+
+PowerShell includes Group Policy settings to help you define consistent option values for servers in
+an enterprise environment. These settings include:
+
+- Console session configuration: Sets a configuration endpoint in which PowerShell is run.
+- Turn on Module Logging: Sets the LogPipelineExecutionDetails property of modules.
+- Turn on PowerShell Script Block Logging: Enables detailed logging of all PowerShell scripts.
+- Turn on Script Execution: Sets the PowerShell execution policy.
+- Turn on PowerShell Transcription: enables capturing of input and output of PowerShell commands
+  into text-based transcripts.
+- Set the default source path for Update-Help: Sets the source for Updatable Help to a directory,
+  not the Internet.
+
+For more information, see
+[about_Group_Policy_Settings](/powershell/module/microsoft.powershell.core/about/about_group_policy_settings).
+
+PowerShell 7 includes Group Policy templates and an installation script in `$PSHOME`.
+
+Group Policy tools use administrative template files (`.admx`, `.adml`) to populate policy settings
+in the user interface. This allows administrators to manage registry-based policy settings. The
+`InstallPSCorePolicyDefinitions.ps1` script installs PowerShell Core Administrative Templates on the
+local machine.
+
+```powershell
+Get-ChildItem -Path $PSHOME -Filter *Core*Policy*
+```
+
+```Output
+    Directory: C:\Program Files\PowerShell\7
+
+Mode                 LastWriteTime         Length Name
+----                 -------------         ------ ----
+-a---           2/27/2020 12:38 AM          15861 InstallPSCorePolicyDefinitions.ps1
+-a---           2/27/2020 12:28 AM           9675 PowerShellCoreExecutionPolicy.adml
+-a---           2/27/2020 12:28 AM           6201 PowerShellCoreExecutionPolicy.admx
+```
+
+### Separate Event Logs
+
+PowerShell 7 includes separate event logs for Windows PowerShell and PowerShell Core. Use the
+following command to get a list of the PowerShell logs.
+
+```powershell
+Get-WinEvent -ListLog *PowerShell*
+```
+
+For more information, see [about_Logging_Windows](/powershell/module/microsoft.powershell.core/about/about_logging_windows).
+
+## Improved editing experience with Visual Studio Code
+
+[Visual Studio Code (VSCode)](https://code.visualstudio.com/) with the
+[PowerShell Extension](https://code.visualstudio.com/docs/languages/powershell) is the supported
+scripting environment for PowerShell 7. The Windows PowerShell Integrated Scripting Environment
+(ISE) only supports Windows PowerShell.
+
+The updated PowerShell extension includes:
+
+- New ISE compatibility module
```

Comment:
Is it a module you're referring to here? Or the "mode"? If the latter, probably go with "mode".

```suggestion
- New ISE compatibility mode
```